style.css

```diff
@@ -1180,3 +1214,132 @@ textarea {
   color: #fff;
   background-color: #2230d2;
 }
+
+@media only screen and (min-width: 768px) {
+  .toggle-hidden1 {
+    display: none;
+  }
+
+  .toggle-hidden2 {
+    display: flex;
+  }
+
+  .headline-container1 {
+    background: url(assets/header-bg-desktop.svg);
+    position: relative;
+    width: 100vw;
+    height: 721.46px;
+    left: 34.6px;
+    top: 0;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .CSS-grid__card-container {
+    background: #e5e5e5;
+
+    /* Works */
+
+    /* Auto layout */
+
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    align-items: center;
+    justify-content: center;
+    padding: 124px auto;
+    gap: 142px;
+    position: relative;
+    width: 100vw;
+    height: 2788px;
+    top: 100px;
+  }
+
+  .CSS-grid__card-container .Card-works {
+    /* Card Works */
+
+    box-sizing: border-box;
+
+    /* Auto layout */
+
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    align-self: center;
+    margin-left: 8%;
+    padding: 16px;
+    gap: 2%;
+    width: 80vw;
+    height: 552px;
+
+    /* Neutrals/N0 */
+
+    background: #fff;
+
+    /* Neutrals/N40 */
+
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
```

Comment:
- [ ] Your work on the card is impressive :+1:, but there are a few adjustments that need to be made to align with the Figma template. Here are the specific changes required:

- Please ensure that the project description ("A daily...") matches the width specified in the template.
- As evident from the template, there is a discrepancy in the background color of the button in comparison to yours.

- [ ] Also, while each work is divided half of it contains the project's picture and the other half contains the project's description :+1:. You should make sure that these switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).

**Your design**

![Screenshot from 2023-03-29 08-09-58](https://user-images.githubusercontent.com/105117832/228548989-f49d3114-f574-4995-a69f-8a6f3c1b46a0.png)

**Figma**

![Screenshot from 2023-03-29 08-10-12](https://user-images.githubusercontent.com/105117832/228549725-28a66c6e-e296-4002-86c0-d2bca2120c0f.png)